org.eclipse.winery.repository/src/main/java/org/eclipse/winery/repository/splitting/Splitting.java

```diff
@@ -400,6 +405,8 @@ public TTopologyTemplate matchingWithDefaultHostSelection (TTopologyTemplate top
 	 * @return modified topology with the replaced Node Templates
 	 */
 	public TTopologyTemplate injectNodeTemplates (TTopologyTemplate topologyTemplate, Map<String, TNodeTemplate> injectNodes) {
+		String id = UUID.randomUUID().toString();
```

Comment:
Please comment, why you chose the approach of a random UUID (and no counter, which you did above).